libs/shared/src/commonProtocol/contractHelpers/Launchpad.ts

```diff
@@ -99,3 +99,19 @@ export const transferLiquidity = async (
     .send({ value: amountIn, from: walletAddress });
   return txReceipt;
 };
+
+// Returns market cap in ETH. Default variables will always return ~29.5 ETH
+// Will need to be converted to USD on the client using APIs used for stake, etc
+// USD Mkt Cap = ETH * USD/ETH rate
+export const getTargetMarketCap = (
+  initialReserve: number = 4.167e8,
+  initialSupply: number = 1e18,
+  currentSupply: number = 4.3e26,
```

Comment:
Out of curiosity, what is `currentSupply` referring to?

Reply:
The current supply at which we want the market cap. Here we're hard coding launchpad liquidity, because we want the price/mkt cap when all tokens have been sold by default. Its variable and not called launchpad liquidity because theoretically this formula/function can and potentially will be used to get mkt cap at any given supply